variation-commons-mongodb/src/main/java/uk/ac/ebi/eva/commons/mongodb/filter/FilterBuilder.java

```diff
@@ -114,21 +115,22 @@ public FilterBuilder withVariantTypes(VariantType variantType) {
 
     public FilterBuilder withAlternates(List<String> alternates) {
         if (alternates != null && !alternates.isEmpty()) {
+            alternates = alternates.stream().map(a->a.toUpperCase()).collect(Collectors.toList());
```

Comment:
Let's not mutate the variable passed in (as a matter of principle). Instead let's just store it in a separate variable...

Reply:
updated